integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchemaCommon.scala

```diff
@@ -121,7 +121,7 @@ case class UpdateTableModel(
     updatedTimeStamp: Long,
     var executorErrors: ExecutionErrors,
     deletedSegments: Seq[Segment],
-    loadAsNewSegment: Boolean = false)
+    loadAsNewSegment: Boolean = true)
```

Comment:
There is no use of horizontal compaction features after this change?
@QiangCai , @marchpure : we need to mark it as deprecated ?

Reply:
remove it, always update into new segment

Reply:
I have removed all code about loadAsNewSegment

Reply:
@marchpure : a) That comment was about horizontal compaction. So, horizontal compaction we need to deprecate or remove was the discussion 
b) loadAsNewSegment you have removed the flag, because can just use insert flow instead of update flow right ?

Reply:
@marchpure :  I have checked all new changes, I saw that you are using horizontal compaction for delete operation only.
a) If any document of horizontal compaction and update document need to update, please handle
b) you can still look up CompactionType.IUD_UPDDEL_DELTA and remove dead codes.

Reply:
I have removed all dead code about  IUD_UPDDEL_DELTA according to your suggestion.
document of horizontal compaction also be updated: the parameter "carbon.horizontal.update.compaction.threshold" is removed.